src/main/kotlin/ecommerce/service/payment/StripeClientService.kt

```diff
@@ -0,0 +1,38 @@
+package ecommerce.service.payment
+
+import com.stripe.model.PaymentIntent
+import com.stripe.param.PaymentIntentCreateParams
+import org.springframework.stereotype.Service
+
+@Service
+class StripeClientService {
```

Comment:
```suggestion
@Component
class StripeClient {
```

Well, I think it comes down to personal preference.
To me, it seems more like a component than a service, since it doesn't have business logic but only handles API calls 🤔

Reply:
Good point! I'll change it to keep the code clear. :)